### ISSUE TITLE
Add support for Brotli compression 

### ISSUE THREAD

**Issue by dutzi** [mkxml/vscode-filesize#25]
It's pretty common these days, and has a port for Node (see [here](https://www.npmjs.com/package/brotli)).

Would be useful to get a glance of the Brotli-compressed size as well within the table:

```
+--------------------------------------------------------------+
| Size         | 2.77 KiB                                      |
|--------------------------------------------------------------|
| Gzipped      | 1008 bytes                                    |
|--------------------------------------------------------------|
| Brotli       | 780 bytes                                     | <--- this guy
|--------------------------------------------------------------|
| Mime type    | application/javascript                        |
|--------------------------------------------------------------|
| Created      | January 31st 2021, 09:51:00                   |
|--------------------------------------------------------------|
| Changed      | January 31st 2021, 11:27:44                   |
+--------------------------------------------------------------+
```

**Comment by dutzi** [comment]
Even better: https://github.com/erwinmombay/brotli-size

**Comment by mkxml** [comment]
Nice, I agree brotli support should be useful. I'm reviewing your PR.

**Comment by mkxml** [comment]
It's available on 3.0.0. Thank you for your contribution!
